runtime/browser/web_view_impl.cc

```diff
@@ -200,7 +201,6 @@ void WebViewImpl::Initialize() {
 
   // TODO(sngn.lee): "protocolhandler,registration,requested"
   //                  custom protocol handler
-  // TODO(sngn.lee): ewk_view_user_media_permission_callback_set
```

Comment:
Is this TODO comment required?
The ewk_view_user_media_permission_callback_set() is called in InitUsermediaCallback() function already.